synapse/tests/test_lib_storm.py

```diff
@@ -97,13 +97,34 @@ def test_storm_alltag(self):
             core.addTufoTag(node, 'foo.bar')
             core.addTufoTag(node, 'baz.faz')
 
+            # test alltag macro syntax
             node = core.eval('#foo.bar')[0]
 
             self.eq(node[1].get('inet:fqdn'), 'vertex.link')
 
             self.nn(node[1].get('#baz'))
             self.nn(node[1].get('#foo.bar'))
 
+            # test alltag macro syntax with limit
+            core.eval('[ inet:fqdn=hehe.com inet:fqdn=haha.com #lol ]')
```

Comment:
this core.eval statement doesn't do anything - should it be wrapped in a self.<test helper> ?